tests/certs/server.csr

```diff
@@ -0,0 +1,17 @@
+-----BEGIN CERTIFICATE REQUEST-----
```

Comment:
Did my old certs not work correctly?

Reply:
I also needed the root CA certificate that signs the server certificate, that's why I needed to rebuild all of the certs